pkg/metrics/prometheus.go

```diff
@@ -26,7 +26,7 @@ var (
 	gitTag                 string
 	webhookInjectionsTotal = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
-			Name: "webhook_injections_total",
+			Name: ":",
```

Comment:
is this relevant c hange ?

Reply:
Definitely not, some vim finger slipping error, I will fix that!